cmake/Packaging.cmake

```diff
@@ -15,10 +15,14 @@ if (ENABLE_PACKAGING)
     set(CPACK_GENERATOR "TGZ")
 
     # if we want to generate all the sub-project packages:
-    #set(CPACK_INSTALL_CMAKE_PROJECTS "${CMAKE_CURRENT_BINARY_DIR}/matio/build;matio;ALL;/" "${CMAKE_CURRENT_BINARY_DIR}/OpenMEEG/build;OpenMEEG;ALL;/")
-
-    # call cpack for OpenMEEG
     set(CPACK_INSTALL_CMAKE_PROJECTS "${CMAKE_CURRENT_BINARY_DIR}/OpenMEEG/build;OpenMEEG;ALL;/")
+    foreach (dep ${MSINTTYPES} zlib ${VTK_} hdf5 matio ${LAPACK})
```

Comment:
I'm merging. I just wonder whether it would not be smart to introduce an OpenMEEG_DEPENDENCIES (or sthg like that) and use it everywhere. Otherwise there is the chance that we change the list in the main CMakeList.txt and not here.....